src/components/fields/PhotoField.tsx

```diff
@@ -42,10 +42,33 @@ const PhotoField = ({
     }
   };
 
+  const enablePhotoDelete = !isOpen && !disabled && !loading;
+
+  const handleOnKeyDown = (e: React.KeyboardEvent) => {
```

Comment:
gal libo naudok ?   ir padarysi tiesiog po to 
`const handleOnKeyDown = useKeyAction(() =>  onImageClick && onImageClick(), disabled);`